tmt/cli.py

```diff
@@ -353,27 +353,28 @@ def show(context, **kwargs):
         echo()
 
 
-@tests.command()
+@tests.command('lint')
 @click.pass_context
 @name_filter_condition
 @click.option(
     '-f', '--fix', is_flag=True,
     help='Attempt to fix all discovered issues.')
 @verbose_debug_quiet
-def lint(context, **kwargs):
+def lint_test(context, **kwargs):
     """
     Check tests against the L1 metadata specification.
 
     Regular expression can be used to filter tests for linting.
     Use '.' to select tests under the current working directory.
     """
-    tmt.Test._save_context(context)
+    # tmt.Test._save_context(context)
     exit_code = 0
     for test in context.obj.tree.tests():
         if not test.lint():
             exit_code = 1
         echo()
-    raise SystemExit(exit_code)
+    #raise SystemExit(exit_code)
```

Comment:
This causes tests to fail because the exit code it not properly reported when single metadata level (e.g. tests only) is linted.